brototype.js

```diff
@@ -140,15 +142,7 @@
         }
     };
 
-    (function() {
-        if (typeof define === 'function' && typeof define.amd === 'object') {
-            define(function() {
-                return Bro;
-            });
-        } else if (typeof module !== 'undefined' && module.exports) {
-            module.exports = Bro;
-        } else if (typeof window !== 'undefined') {
-            window.Bro = Bro;
-        }
-    })();
```

Comment:
nearly done with example tests for ya.  Sorry it took so long, life's been crazy busy.